src/main/java/org/apache/accumulo/access/AccessEvaluator.java

```diff
@@ -88,12 +90,12 @@ static AccessEvaluator of(Authorizations authorizations) {
   }
 
   /**
-   * Creates an AccessEvaluator from an Authorizer object
+   * Creates an AccessEvaluator from a Predicate&lt;String&gt;
```

Comment:
I wonder if we need a little more verbiage to explain when the predicate should return true vs false. Given that this is part of the public API we may want to explain that the AccessEvaluator implementation is going to call `test` passing in a single Authorization and it should return true if the exact String matches.

Reply:
Added to the javadoc in 4e80d25 to try to address this.